apps/rotm/behaviours/skip-step.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const isIncluded = (a, b) => a.some(v => b.includes(v));
+
+module.exports = (config) => {
+  return superclass => class extends superclass {
+    getValues(req, res, callback) {
+      super.getValues(req, res, (err, values) => {
+
+        if (config && config.steps) {
+          let completedSteps = values.steps;
+          const currentStep = req.form.options.route;
+          if (config.steps.includes(currentStep) === false) {
```

Comment:
I assume this is `Array.prototype.includes` and not `String.prototype.includes`?
If you are using `includes` coerce `config.steps` to an `array` to ensure no weirdness in the case where a user passes `steps` as a `string`.

Reply:
Or just use `indexOf`